exec.go

```diff
@@ -45,7 +45,7 @@ func sudoLoop() {
 
 func updateSudo() {
 	for {
-		err := show(exec.Command("sudo", "-v"))
+		err := show(exec.Command(config.SudoBin, "-v"))
```

Comment:
Shouldnt sudoflags also be used here?

Reply:
Well, I thought about it, but I don't know, the only purpose of that function is to refresh the sudo user's cached credentials. But thinking about it know, I guess the SudoFlags should be here too, you're right.

Reply:
Seems there's no reason not to. May want to make a wrapper around sudo in exec.go.